client/components/LearnerViewDirective.js

```diff
@@ -249,10 +249,20 @@ tie.directive('learnerView', [function() {
         $scope.submitCode = function(code) {
           SolutionHandlerService
             .processSolutionAsync(prompts[currentPromptIndex], code, language)
-            .then(appendFeedback);
+            .then(setFeedback);
         };
 
-        clearFeedback();
+        $scope.feedbackParagraphs = [
+          'Greetings!',
+          'This set of exercises focuses on string manipulation.',
```

Comment:
Avoid including exercise-specific content within the view directive? I.e., to me, it appears this line belongs somewhere else, like i18n.js